test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishTrimmedWindowsFormsAndWPFApps.cs

```diff
@@ -358,6 +366,11 @@ private void InitializeComponent()
         this.Text = ""Form1"";
     }
     #endregion
+
+    protected override void OnShown(EventArgs e)
+    {
+        Close();
```

Comment:
To exit with specific code I believe we change this to call `Environment.Exit()` with preferred exit code would work

Reply:
`%errorlevel%` seems still to be 0.